### ISSUE TITLE
About model performance

### ISSUE THREAD

**Issue by LordSlime** [yfw/nlotm#1]
Hi, authors, this is a great work. It is impressive that you extract factors utilizing multiple codebooks and slot attention.
However, I encountered some problem during training. I followed the instruction using CLEVER-easy datasets but got bad performance, especially in attention map part. 
<img width="912" height="1042" alt="Image" src="https://github.com/user-attachments/assets/a0a2ad62-1fce-43c9-9119-decb858189b6" />
The attention map didn't separate objects successfully and block representation didn't learn enough factor-level representation.
My training environment is pytorch 2.3.1 with cuda 12.1, since cuda 11.8 is not compatible with my hardware.
Do you have any solutions to these problems?

**Comment by yfw** [comment]
Hi just want to double check if you are using `lr_enc=5e-5`? We had found this worked better with pytorch 2.3.1 than what was reported in the paper. Besides that it may help to try other encoder learning rates since it is a bit sensitive to this hyperparameter.

**Comment by LordSlime** [comment]
Yes, I tried setting encoder learning rate as 1e-4 and 5e-5. However, it did not work. The attention map just couldn't segment objects well.
